src/components/NavBar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { changeFilter } from '../actions/books';
+import CategoryFilter from './CategoryFilter';
+
+const NavBar = () => {
+  const dispatch = useDispatch();
+  const handleFilterChange = (event) => {
+    dispatch(changeFilter(event.target.value));
+  };
+
+  // The main-1 class was initially main, it may not be needed
+  return (
+    <div className="navbar main-1">
+      <div className="navbar content">
+        <h1>BookStore CMS</h1>
+        <h3>BOOKS</h3>
+        <h3>CATEGORIES</h3>
+        <CategoryFilter handleFilterChange={handleFilterChange} />
+        <span><i className="fas fa-user-alt" /></span>
```

Comment:
The Avatar should be on the same line with the button `ADD BOOK`. kindly implement it, I have tested it with three different browser gotten the same result